overlays/custom-packages/pulseaudio/default.nix

```diff
@@ -0,0 +1,7 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{ prev, ... }:
+prev.pulseaudio.overrideAttrs (_prevAttrs: {
```

Comment:
Btw. we switched now to pipewire in all NixOS desktop environments. Historically also bluetooth support is better there (more codecs). And it doesn't suffer from other issues i.e. mute audio after suspend.

Reply:
Ah. I see pipewire being used above. So this is only for applications.